dev/Microsoft.Process.Environment/Microsoft.Process.Environment.def

```diff
@@ -0,0 +1,3 @@
+EXPORTS
+DllCanUnloadNow = WINRT_CanUnloadNow                    PRIVATE
+DllGetActivationFactory = WINRT_GetActivationFactory    PRIVATE
```

Comment:
You should be able to delete this file - the .def for the Project Reunion DLL already has these.